cmd/generateSott/generateSott.go

```diff
@@ -39,16 +37,19 @@ func NewgenerateSottCmd() *cobra.Command {
 		Use:     "generate-sott",
 		Short:   "generates sott",
 		Long:    `This commmand generates sott`,
-		Example: heredoc.Doc(`$ lr generate sott`),
+		Example: heredoc.Doc(`$ lr generate-sott`),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if opts.FromDate == "" {
-				return &cmdutil.FlagError{Err: errors.New("`FromDate` is require argument")}
-			}
-			if opts.ToDate == "" {
-				return &cmdutil.FlagError{Err: errors.New("`ToDate` is require argument")}
-			}
-			if opts.Technology == "" {
-				return &cmdutil.FlagError{Err: errors.New("`technology` is require argument")}
+			if opts.FromDate == "" || opts.ToDate == "" || opts.Technology == "" {
```

Comment:
Why we need this condition `if opts.FromDate == "" || opts.ToDate == "" || opts.Technology == "" {`